src/test/java/org/casbin/jcasbin/main/EnforcerUnitTest.java

```diff
@@ -398,6 +406,54 @@ public void testInitWithAdapter() {
         testEnforce(e, "bob", "data2", "write", true);
     }
 
+    @Test
+    public void testFileAdapterAutoSave() {
+        Enforcer e = new Enforcer("examples/basic_model.conf", "examples/basic_policy.csv");
+
+        // test addPolicy() autoSave
+        e.enableAutoSave(false);
```

Comment:
For this CSV adapter, set AutoSave to false by default. Only call `e.enableAutoSave(true)` to enable